introduction.md

```diff
@@ -1 +1,5 @@
-I'm a software developer. HTML is my favorite code school path
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?